attributes/vc10.rb

```diff
@@ -26,9 +26,9 @@
 
 default['vcruntime']['vc10']['x86']['10.0.40219']['url']       = 'http://download.microsoft.com/download/C/6/D/C6D0FD4E-9E53-4897-9B91-836EBA2AACD3/vcredist_x86.exe'
 default['vcruntime']['vc10']['x86']['10.0.40219']['sha256sum'] = '66b797b3b4f99488f53c2b676610dfe9868984c779536891a8d8f73ee214bc4b'
-default['vcruntime']['vc10']['x86']['10.0.40219']['name']      = 'Microsoft Visual C++ 2010 x86 Redistributable - 10.0.40219'
+default['vcruntime']['vc10']['x86']['10.0.40219']['name']      = 'Microsoft Visual C++ 2010  x86 Redistributable - 10.0.40219'
```

Comment:
yes the double space was meant to be included, i copied directly from the registry the entry to ake sure it was detecting the correct entry in windows